visual-tests/scripts/visual-tests-build.mjs

```diff
@@ -0,0 +1,72 @@
+import path from 'path';
+import execa from 'execa';
+import fse from 'fs-extra';
+
+// Main wrapper should be defined as a first one -
+// it will be used to generate golden screenshots,
+// rest of wrappers will be compared to it.
+const wrappers = ['js', 'angular-13', 'react'];
+// const wrappers = ['js', 'angular-13', 'react', 'vue'];
+const baseBranch = 'develop';
+const currentBranchName = (await execa.command('git rev-parse --abbrev-ref HEAD', { silent: true })).stdout;
+const dirs = {
+  init: process.cwd(),
+  examples: '../examples/next/visual-tests',
+  codeToRun: 'demo',
+};
+
+// const HOTversion = '12.2.0';
+
+process.chdir(dirs.examples);
+execa.command('npm install');
+
+// If we are on a base branch, we do not want to run all of tests
+// and make screenshots for all of wrappers.
+// On base branch we need to create golden screenshots from main wrapper only,
+// which is declared as a first position in wrappers array.
+
+for (let i = 0, maxi = (currentBranchName === baseBranch ? 1 : wrappers.length); i < maxi; ++i) {
+  process.chdir(`${wrappers[i]}/${dirs.codeToRun}`);
+
+  // eslint-disable-next-line no-await-in-loop
+  await execa.command('npm install', { stdout: 'inherit' });
+  // eslint-disable-next-line no-await-in-loop
+  await execa.command('npm run build', { stdout: 'inherit' });
+  // eslint-disable-next-line no-await-in-loop
+  const localhostProcess = execa.command('npm run start', {
+    detached: true,
+    stdio: 'ignore',
+    windowsHide: true });
+
+  process.chdir(dirs.init);
```

Comment:
Just a suggestion - maybe we could utilize `execa`'s cwd option to run the scripts from different directories?
I'm afraid that changing them externally could potentially lead to some problems - for example, running a script from a wrong directory when the directory was mistakenly not changed after the previous script finished.

Reply:
hi! your suggestion has been implemented